### DISCUSSION TITLE
Beatmap cards show last update time with UTC offset set to 0

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#16103] [Q&A] [answered]
As the images show, all beatmap cards have the last update time with `UTC+0`.
But if you go to check in the beatmapset overlay you can see that it should be, in my case, `UTC+1`/`+2`:

![1](https://user-images.githubusercontent.com/43073074/146229329-85de83c8-4001-426b-ae7d-463e77db96dc.png)

![2](https://user-images.githubusercontent.com/43073074/146229491-7e2954e0-caf6-4185-8340-b70bd62491eb.png)

---

![3](https://user-images.githubusercontent.com/43073074/146230738-26ec2b1e-2acf-4259-ba64-085466262af9.png)

![4](https://user-images.githubusercontent.com/43073074/146230785-1ab15e40-9e1a-4ca6-ba42-c5ab49023bd4.png)

(In case you're wondering, on web it seems to show the correct offset)

![5](https://user-images.githubusercontent.com/43073074/146231478-946e171a-f5ee-46a9-8224-6f653c055352.png)

**Comment by frenzibyte** [comment, marked as answer]
Escalated to #16104
